client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -165,9 +166,12 @@ var DomainSearch = React.createClass( {
 
 module.exports = connect(
 	function( state, props ) {
+		const selectedSite = props.sites.getSelectedSite();
```

Comment:
I didn't really introduced it, simply shared it with `sitePlans` below. Using `getSelectedSite` here would be better indeed, but it requires more change like removing the `sites` from props and then removing mixins... I'll try to do it for this file, it will indeed look better 👍